lib/extensions/conditional/active_record/base.rb

```diff
@@ -2,16 +2,19 @@ module Extensions::Conditional::ActiveRecord::Base
   module ClassMethods
     # Functions from conditional-and-condition framework.
     # Declare this function in the conditional model that requires conditions.
-    def acts_as_conditional
+    #
+    # @param [Hash] options Extra options to pass to the association declaration
+    def acts_as_conditional(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `acts_as_conditional(*)` if you want the method to accept any arguments but don't care about them.